ui/apps/everest/.e2e/release/restore-new-cluster.e2e.ts

```diff
@@ -0,0 +1,534 @@
+        await page.getByTestId(`add-db-cluster-button-${db}`).click();
+
+        await test.step('Populate basic information', async () => {
+          await populateBasicInformation(
+            page,
+            namespace,
+            clusterName,
+            db,
+            storageClasses[0],
+            false
+          );
+          await moveForward(page);
+        });
+
+        await test.step('Populate resources', async () => {
+          await page
+            .getByRole('button')
+            .getByText(size + ' node')
+            .click();
+
+          await expect(page.getByText('Nodes (' + size + ')')).toBeVisible();
+          await populateResources(page, 0.6, 1, 1, size);
+          await moveForward(page);
+        });
+
+        await test.step('Populate backups', async () => {
+          await moveForward(page);
+        });
+
+        await test.step('Populate advanced db config', async () => {
+          await populateAdvancedConfig(page, db, '', true, '');
+          await moveForward(page);
+        });
+
+        await test.step('Populate monitoring', async () => {
+          await page.getByTestId('switch-input-monitoring').click();
+          await page
+            .getByTestId('text-input-monitoring-instance')
+            .fill(monitoringName);
+          await expect(
+            page.getByTestId('text-input-monitoring-instance')
+          ).toHaveValue(monitoringName);
+        });
+
+        await test.step('Submit wizard', async () => {
+          await submitWizard(page);
+        });
+
+        await test.step('Check db list and status', async () => {
+          await page.goto('/databases');
+          await waitForStatus(page, clusterName, 'Initializing', 15000);
+          await waitForStatus(page, clusterName, 'Up', 600000);
+        });
+
+        await test.step('Check db cluster k8s object options', async () => {
+          const addedCluster = await getDbClusterAPI(
+            clusterName,
+            EVEREST_CI_NAMESPACES.EVEREST_UI,
+            request,
+            token
+          );
+
+          expect(addedCluster?.spec.engine.type).toBe(db);
+          expect(addedCluster?.spec.engine.replicas).toBe(size);
+          expect(['600m', '0.6']).toContain(
+            addedCluster?.spec.engine.resources?.cpu.toString()
+          );
+          expect(addedCluster?.spec.engine.resources?.memory.toString()).toBe(
+            '1G'
+          );
+          expect(addedCluster?.spec.engine.storage.size.toString()).toBe('1Gi');
+          expect(addedCluster?.spec.proxy.expose.type).toBe('internal');
+          if (db != 'psmdb') {
+            expect(addedCluster?.spec.proxy.replicas).toBe(size);
+          }
+        });
+      });
+
+      test(`Add data [${db} size ${size}]`, async () => {
+        await prepareTestDB(clusterName, namespace);
+      });
+
+      test(`Create backup schedules [${db} size ${size}]`, async ({ page }) => {
```

Comment:
Please use on-demand backups instead of scheduled backups, because that way we don't need to wait for backups to succeed in tests (much faster) and I don't see how restore from scheduled backup is different than restore from on demand full backup.